### DISCUSSION TITLE
Can Finn be used with the rfsoc4x2 board?

### DISCUSSION THREAD

**Discussion by harrisonasd** [Xilinx/finn#982] [General]
Can Finn be used with the rfsoc4x2 board?
I didn't see support for the rfsoc4x2 board on the Finn official website.
but I did see a paper where the rfsoc2x2 was used to implement Finn

**Comment by fpjentzsch** [comment]
Both boards should be directly supported since this PR: https://github.com/Xilinx/finn/pull/787

**Reply by harrisonasd** [reply]
thank you very much
